data_tools/compute_coding_progress.py

```diff
@@ -6,36 +6,37 @@
 
 if (len(sys.argv) != 2):
     print ("Usage python compute_coding_progress.py coda_crypto_token")
-    exit(1)
+        exit(1)
 
 CODA_CRYPTO_TOKEN_PATH = sys.argv[1]
-
 fcw.init_client(CODA_CRYPTO_TOKEN_PATH)
-data = {}
-
-ids = fcw.get_dataset_ids()
-
-data['coding_progress'] = {}
```

Comment:
This change turns this code into a function - great - but it will break being able to run this tool from the command line. That's a bit of a problem.

Please can you add a __main__ method to the bottom of the file that calls the functions, so that the functions can be used either directly on the command line, or as functions

Reply:
Thanks for the change